tools/ratbagctl.in

```diff
@@ -1540,21 +1540,28 @@ class RatbagParserRoot(object):
         self.parser.print_help()
 
 
+def open_ratbagd():
+    try:
+        r = Ratbagd()
+    except RatbagdDBusUnavailable:
+        print("Unable to connect to ratbagd over dbus")
+        return None
+    else:
+        if not r.devices:
+            print("No devices available.")
+
+        return r
+
```

Comment:
Well, given the amount of flake8 errors this series introduce, I might as well add such a check before messing up too much in the python code.